### PR TITLE
Update documents, color properties descriptions

### PR DESCRIPTION
**Pull request by manishmz**
Updated layer documents color properties descriptions

### CODE REVIEW

**Comment by 1chandu** [comment]
Instead of updating the existing text, I think it is good to just add one line specifying `a` affects the transparency:
```
`a` affects the transparency, default value `255` when supplying custom value for `a`, lower value makes it transparent and higher value makes it opaque
```

File: docs/layers/arc-layer.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Method called to retrieve the target position of each object.
 
 * Default: `[0, 0, 0, 255]`
 
-The rgba color at the source, in `r, g, b, [a]`. Each component is in the 0-255 range.
+The rgba color at the source, is `[r, g, b, [a]]`. `r, g, b, a` is in 0-255 range and `a` is 255 by default.
```
Comment (Pessimistress) [inline docs/layers/arc-layer.md]: `...in the format of [r, g, b, [a]]. Each channel is a number between 0-255...`

Comment (1chandu) [reply docs/layers/arc-layer.md]: @manishmz can you make this change and push?

**Comment by 1chandu** [comment]
Closing in favor of : #3993

**Pull request closed without merging**